src/noggit/ui/RotationEditor.cpp

```diff
@@ -44,15 +44,15 @@ namespace noggit
       layout->addRow(new QLabel("- rotation and scale only\n  change when pressing enter", this));
       layout->addRow(new QLabel("- scaling is multiplicative", this));
 
-      _rotation_x->setRange (-180.f, 180.f);
+      _rotation_x->setRange (-1800.f, 1800.f);
```

Comment:
This feels like it makes little sense: Before it was possible to rotate 360°, now it is possible to 3600°?!

Reply:
The issue is that the range of -180/+180 doesn't actually make sense with the numbers noggit ends up using under the hood, I couldn't get around to fix that, but you end up with numbers above 360 or below -180, which still get saved correctly but the interface just shows as the min or max, hiding the real value. The correct fix would be to loop the number correctly but I didn't find the depths that would have to be hooked in.